### PR TITLE
Shift from browserstack-webdriver to selenium-webdriver

### PR DESCRIPTION
**Pull request by Varun-garg**
browserstack-webdriver is still at selenium 2.xx, time to depreciate it.

### CODE REVIEW

**Comment by Varun-garg** [comment]
Changes applied @francisf

File: mocha/BrowserStackRemoteUnit.js [hunk 1]
```diff
@@ -1,34 +1,49 @@
...
+          }));
+        });
       });
-    }, 1000);
+    });
   });
 
-  test.after(function() { driver.quit(); });
+  after(function () {
+    driver.quit();
```
Comment (francisf) [inline mocha/BrowserStackRemoteUnit.js]: if driver

File: BrowserStack.js [hunk 2]
```diff
@@ -1,22 +1,36 @@
...
+	os: 'Windows',
+	build: 'Test Build 01',
+	project: 'My Awesome App',
+	'browserstack.debug': true,
+};
+
+let driver = new Builder()
+	.usingHttpAgent(HttpAgent)
+	.withCapabilities(capabilities)
+	.usingServer(`http://${BROWSERSTACK_USERNAME}:${BROWSERSTACK_ACCESS_KEY}@hub-cloud.browserstack.com/wd/hub`)
```
Comment (francisf) [inline BrowserStack.js]:
```suggestion
	.usingServer(`https://${BROWSERSTACK_USERNAME}:${BROWSERSTACK_ACCESS_KEY}@hub-cloud.browserstack.com/wd/hub`)
```

File: BrowserStack.js [hunk 3]
```diff
@@ -1,22 +1,36 @@
...
+	driver.findElement(By.name('q')).then((element) => {
+		element.sendKeys('BrowserStack', Key.RETURN).then(() => {
+			driver.wait(until.titleContains('BrowserStack')).then(() => driver.getTitle().then((title) => {
+				console.log(title);
+				driver.quit();
+			}));
+		})
 	});
-}, 1000);
+});
```
Comment (francisf) [inline BrowserStack.js]: for now lets go ahead with this. need to figure out better handling of finally with chained promises.

**Pull request merged**